docs/how-to-build.md

```diff
@@ -4,244 +4,232 @@ slug: /how-to-build
+   ```shell
+   sudo usermod -aG docker $USER
    ```
 
-  This creates `gravitino-iceberg-rest-server-{version}.tar.gz` and `gravitino-iceberg-rest-server-{version}.tar.gz.sha256` under the `distribution` directory. You can uncompress and deploy it to use the Gravitino Iceberg REST server.
-
-## How to build Apache Gravitino on Windows (Using WSL)
+## Build Gravitino
 
-### Download WSL (Ubuntu)
-
-**On Windows:**
-
-Refer to this guide for installation: [WSL Installation Guide](https://learn.microsoft.com/en-us/windows/wsl/install)
-
-*Note: Gravitino can run successfully on Ubuntu 22.04*
-
-This step involves setting up your Windows machine's Windows Subsystem for Linux (WSL). WSL allows you to run a Linux distribution alongside Windows, providing a Linux-like environment for development.
-
-### Update package list and install necessary packages
-
-**On Ubuntu (WSL):**
-
-```shell
-sudo apt update
-sudo apt install apt-transport-https ca-certificates curl software-properties-common
-```
+1. Clone the Gravitino project.
 
-Updating the package list ensures you have the latest information on the newest versions of packages and dependencies. Installing the necessary packages lets your system download and manage additional software securely.
+   ```shell
+   git clone git@github.com:apache/gravitino.git
+   ```
 
-### Download and setup Java SDK 17 (11 or 8 also works)
+1. Build the Gravitino project. Running this for the first time can take 15 minutes or more.
 
-**On Ubuntu (WSL):**
+   ```shell
+   cd gravitino
+   ./gradlew build
+   ```
 
-1. Edit your `~/.bashrc` file using any editor. Here, `vim` is used:
+   You can customize the property `jdkVersion` in the `gradle.properties` file to use JDK 11 or JDK 17.
+   Alternatively, you can specify the version using the `-P` Gradle flag. e.g.
 
    ```shell
-   vim ~/.bashrc
+   ./gradlew build -PjdkVersion=11
    ```
 
-2. Add the following lines at the end of the file. Replace `/usr/lib/jvm/java-11-openjdk-amd64` with your actual Java installation path:
-
-   ```sh
-   export JAVA_HOME="/usr/lib/jvm/java-11-openjdk-amd64"
-   export PATH=$PATH:$JAVA_HOME/bin
+   The `./gradlew build` command builds all the Gravitino components,
+   including the Gravitino server, Java and Python clients, Trino and Spark connectors etc.
+ 
+   You can customize the `pythonVersion` property in the `gradle.properties` file
+   to use Python 3.9, 3.10, or 3.11. Alternatively, you can set it on the command line
+   using the `-P` Gradle flag. e.g.
+ 
+   ```shell
+   ./gradlew build -PpythonVersion=3.9
    ```
 
-3. Save and quit in vim using `:wq`.
+   :::note
+   The Gravitino libraries built are Java 8 compatible and verified under Java 8, 11, and 17.
+   You can use JRE 8, 11 or 17 to run the Gravitino server,
+   no matter which JDK version was used to build the project.
+ 
+   The built JARs are under the modules `build/libs` directory.
+   You can publish them in the Maven repositor for your projecty
```

Comment:
Breaking sentences over lines has hidden errors here.
"You can run the Gravitino server using JRE 8, 11, or 17, regardless of which JDK version was used to build the project."
"to the Maven repository for your project." or "to your project's Maven repository."

Reply:
Sorry for the typo.